draft-ietf-httpbis-cache-latest.xml

```diff
@@ -1480,6 +1483,15 @@
 </t>
 </section>
 
+<section title="no-store" anchor="cache-response-directive.must-understand">
+   <iref item="must-understand (cache directive)" primary="true" />
+<t>
+   The "must-understand" response directive indicates that a cache &MUST; 
+   understand and implement the caching semantics of the response to be able
+   to store it; if it does not, it &MUST-NOT; store the response.
+</t>
```

Comment:
I'd replace the second bcp14 keyword by prose; it's the same requirement, no?

Reply:
Fixed.